### ISSUE TITLE
SA1303: Can we configure this to warn if the 1st char is _

### ISSUE THREAD

**Issue by TickettEnterprises** [DotNetAnalyzers/StyleCopAnalyzers#3793]
For Private Const we want the rule to allow:
```csharp
private const string Test = "Test String"
```

But not allow:
```csharp
private const string _test = "test String"
private const string _Test = "Test String""
private const string test = "Test string" (Which the rule already does)
```

We are aware we could turn on the rule SA1309: Field names should not begin with underscore.
But we want private field names to use _

For example: 
```csharp
private string _titleString = "Page Title"
```

Is there away to configure this behaviour with existing rules?

**Comment by sharwell** [comment]
What about turning on [SA1300](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SA1300.md) and [SX1309](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SX1309.md)?

**Comment by TickettEnterprises** [comment]
> What about turning on [SA1300](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SA1300.md) and [SX1309](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SX1309.md)?

This would not work because what we are trying to achieve is:

Private const use PascalCase starting with capital.
But we still want none const privates to start with a _

Turning on [SX1309](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SX1309.md) would mean all fields need to start with _.

**Comment by sharwell** [comment]
The code for SX1309 seems to suggest otherwise:

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/StyleCop.Analyzers/StyleCop.Analyzers/NamingRules/SX1309FieldNamesMustBeginWithUnderscore.cs#L68-L71

**Comment by TickettEnterprises** [comment]
Have checked the configuration and both [SA1300](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SA1300.md) and [SX1309](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SX1309.md) are set to error.

However no error is shown for:

```chasrp
private const string _testing = "";
```

Only shown for 
```csharp
private const string _testing = "";
private string testing = "";
```

**Comment by TickettEnterprises** [comment]
> Have checked the configuration and both [SA1300](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SA1300.md) and [SX1309](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SX1309.md) are set to error.
> 
> However no error is shown for:
> 
> ```
> private const string _testing = "";
> ```
> 
> Only shown for
> 
> ```cs
> private const string _testing = "";
> private string testing = "";
> ```

This is not the intended result, as outlined we want to enforce private const to start with Cap and not _.
And private to start with a _
